src/main/java/uk/gov/justice/digital/delius/controller/secure/OffendersResource.java

```diff
@@ -363,5 +364,29 @@ public Conviction getConvictionForOffenderByCrnAndConvictionId(
             .orElseThrow(() -> new NotFoundException(String.format("Offender with crn %s not found", crn)))
             .orElseThrow(() -> new NotFoundException(String.format("Conviction with ID %s for Offender with crn %s not found", convictionId, crn)));
     }
+
+    @ApiOperation(value = "Return the breaches for a conviction ID and a CRN")
+    @ApiResponses(
+        value = {
+            @ApiResponse(code = 200, message = "OK", response = Conviction.class),
```

Comment:
Little typo on response. In fact there has been a suggestion in DPS that we stop adding the 200 response annotation since apparently it will be generated automatically anyway, and always with the correct type, so may as well delete this line.

Reply:
agreed, please remove the 200.  Worth also firing up the application and looking at the swagger docs to check they make sense e.g. duplicate `crn` documented below

Reply:
OK - done